src/main/groovy/jenkins/automation/utils/BaseJobBuilder.groovy

```diff
@@ -33,7 +33,11 @@ class BaseJobBuilder {
                 numToKeep(10)
             }
             publishers{
-                mailer emails.join(' ')  //TODO use extended email
+                allowBrokenBuildClaiming()
+
+                if(emails){
```

Comment:
can pass an empty array? This guard statements is not enough. If it is in the base job, we should require or default it. 
Otherwise take it out of the base job.